landingpage/src/components/TransactionTool.tsx

```diff
@@ -0,0 +1,175 @@
+import { /*Transaction,*/ Psbt, script } from 'bitcoinjs-lib'
+import { bitcoinExplorer, PlebNameData, PlebNameHistory, util } from 'plebnames'
+import MarkedTextWithCopy from './MarkedTextWithCopy'
+import { useEffect, useState } from 'react'
+import DropDownContent from './HeaderWithSearch/DropDownContent'
+import AlterConfigForName, { AlterOptionKey } from './AlterConfigForName'
+
+interface TransactionToolProps {
+	mode: 'claimAndInscribe'|'inscribe'
+	name: string;
+	history?: PlebNameHistory;
+}
+
+type InscriptionMap = Partial<Record<AlterOptionKey, string>>;
```

Comment:
what do you think about just `type InscriptionsInAscii = string[];` or `type Inscriptions = {ascii: string, valid: boolean}[]`

Reply:
I find it more clear to have a key/value structure because the Tx Tool needs to bookkeep added and changed inscriptions.